templates/JazzyFile.stencil

```diff
@@ -0,0 +1,22 @@
+author: Microsoft
+author_url: https://azure.github.io/azure-sdk/
+github_url: https://github.com/Azure/azure-sdk-for-ios
+module: {{ model.name }}
+module_version: 0.1
+readme: ../sdk/{{ model.namespace }}/{{ model.name }}/README.md
+skip_undocumented: false
+hide_unlisted_documentation: false
+theme: fullwidth
+output: ../build/jazzy/{{ model.name }}
+clean: true
+sdk: iphonesimulator
+xcodebuild_arguments:
+  - "-workspace"
+  - "AzureSDK.xcworkspace"
+  - "-scheme"
+  - "{{ model.name }}"
+  - "-sdk"
+  - "iphonesimulator"
+exclude:
+  - "*/Source/ObjectiveCCompatibility/*"
```

Comment:
we probably don't need these 2 excludes , right?

Reply:
I kept them in because they were taken from storage.